db/migrate/20210808184948_limit_activity_characters.rb

```diff
@@ -0,0 +1,11 @@
+class LimitActivityCharacters < ActiveRecord::Migration[6.0]
+  def up
+    Activity.where('length(name) > 52').each {|activity|
```

Comment:
geen down migratie, terwijl dat wel vrij makkelijk is (limit terugveranderen, activiteitnamen gelijkhouden)